errors/Makefile.am

```
@@ -53,7 +53,7 @@ translate-warn:
 
 $(TRANSLATE_LANGUAGES): $(ERROR_TEMPLATES)
 
-NOTIDY=`$(PO2HTML) --help | grep -o "\-\-notidy"`
+NOTIDY=`$(PO2HTML) --help | grep -o "\--notidy"`
```

While grep has [added](https://lists.nongnu.org/archive/html/bug-gnulib/2022-06/txtL0fZB_K0xd.txt)  special code to suppress the stray backslash warning in the beginning of the regex, I think we should stop escaping what should not be escaped instead of relying on that special handling:

```suggestion
NOTIDY=`$(PO2HTML) --help | grep -o "[-]-notidy"`
```

or

```suggestion
NOTIDY=`$(PO2HTML) --help | grep -o -- "--notidy"`
```

or

```suggestion
NOTIDY=`$(PO2HTML) --help | grep -o -e "--notidy"`
```

---

Going with the "[-]" form since the ```-e``` and ```--``` are not universally accepted by non-GNU grep implementations.